exercises/space-age/example.nim

```diff
@@ -0,0 +1,38 @@
+import
+    math
+
+type
+    SpaceAge* = object
```

Comment:
Small style and convention nitpicks:

* Use 2 spaces
* Use camelCase

Reply:
@dom96  Just checking, but user-defined types should start with a capital, right? So `SpaceAge`, is fine.

I fixed my editor to use 2 spaces now, so that should fix this issue going forward :) Thanks for pointing this out to me!

Reply:
Yup, I was right based on the Style Guide, https://nim-lang.org/docs/nep1.html#introduction-naming-conventions So, `SpaceAge` is fine. Thanks again for the help!